migrations/migrations/20191113182159-create_users.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    queryInterface.createTable('users', {
+      id: { type: Sequelize.INTEGER, allowNull: false, autoIncrement: true, primaryKey: true, unique: true },
+      first_name: { type: Sequelize.STRING, allowNull: false },
+      last_name: { type: Sequelize.STRING, allowNull: false },
+      email: { type: Sequelize.STRING, allowNull: false, unique: true },
+      password: { type: Sequelize.STRING, allowNull: false }
```

Comment:
You did not use the `npx sequelize-cli model:generate` command, did you? The metadata I told you about (created_at, updated_at) should be here as well. 
That command creates a create migration and a model for you, which includes the metadata. They'll be in camelCase, but you can manually change them to snake case.